packages/abseil-cpp/build.sh

```diff
@@ -2,15 +2,15 @@ TERMUX_PKG_HOMEPAGE=https://abseil.io/
 TERMUX_PKG_DESCRIPTION="Abseil C++ Common Libraries"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-# Do not forget to rebuild revdeps along with EVERY "major" version bump.
-TERMUX_PKG_VERSION="20240116.2"
-TERMUX_PKG_REVISION=1
+# Do not forget to rebuild revdeps and fix version in their TERMUX_PKG_DEPENDS along with EVERY "major" version bump.
+TERMUX_PKG_VERSION="20250127.0"
 TERMUX_PKG_SRCURL=https://github.com/abseil/abseil-cpp/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=733726b8c3a6d39a4120d7e45ea8b41a434cdacde401cba500f14236c49b39dc
+TERMUX_PKG_SHA256=16242f394245627e508ec6bb296b433c90f8d914f73b9c026fddb905e27276e8
 # updating this will break libprotobuf
 TERMUX_PKG_AUTO_UPDATE=false
 TERMUX_PKG_DEPENDS="libc++"
 TERMUX_PKG_CONFLICTS="libgrpc (<< 1.52.0-1)"
+TERMUX_PKG_BREAKS="termux-gui-c (<= 0.1.3-1), python-pyarrow (<= 19.0.0), python-onnxruntime (<= 1.20.2), python-grpcio (<= 1.70.1), protobuf-static (<= 25.1-3), mosh (<= 1.4.0-11), mosh-perl (<= 1.4.0-11), libwebrtc-audio-processing (<= 1.3-1), libre2 (<= 2024.07.02-1), libprotobuf-c (<= 1.5.1), libprotobuf (<= 2:25.1-3), libutf8-range (<= 2:25.1-3), protobuf-dev (<= 2:25.1-3), protobuf (<= 2:25.1-3), libncnn (<= 20230627-6), libgrpc (<= 1.70.1), libarrow-cpp (<= 19.0.0), et (<= 6.2.9), android-tools (<= 35.0.2)"
```

Comment:
Yeah I don't know how I feel about this one.
That's gonna be a pain in the ass to keep updated and as Thunder already mentioned,
we explicitly do not support partial upgrades already so this smells.

Reply:
Ok, I'll remove it.

Reply:
But what about version requirement in TERMUX_PKG_DEPENDS, should I keep it?

Reply:
I'd say no.
Something akin to an SOVERSION guard in Abseil's build script might be a more sustainable solution.

That raises the question though, what constitutes a "major" version for Abseil?

Reply:
I do not think there is one after they switched to date-based versioning.

Reply:
We'd need to do some branch parsing by the looks of it.
https://abseil.io/about/releases